### PR TITLE
Next major

### PR DESCRIPTION
**Pull request by colebemis**

### CODE REVIEW

File: .changeset/deprecate-actionlistv1-promote-actionlistv2.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+'@primer/react': major
+---
+
+### ActionList
```
Comment (colebemis) [inline .changeset/deprecate-actionlistv1-promote-actionlistv2.md:5]: I think we might need a linebreak here

CleanShot 2022-02-28 at 14 03 01



```suggestion
<br />

### ActionList
```

File: .changeset/deprecate-actionmenuv1-promote-actionmenuv2.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+'@primer/react': major
+---
+
+### ActionMenu
```
Comment (colebemis) [inline .changeset/deprecate-actionmenuv1-promote-actionmenuv2.md:5]:
```suggestion
<br />

### ActionMenu
```

File: .changeset/deprecate-dropdownmenu.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+---
+'@primer/react': major
+---
+
+### DropdownMenu
```
Comment (colebemis) [inline .changeset/deprecate-dropdownmenu.md:5]:
```suggestion
<br />

### DropdownMenu
```

File: .changeset/empty-pillows-hunt.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
+---
+'@primer/react': major
+---
+
+### ActionList
```
Comment (colebemis) [inline .changeset/empty-pillows-hunt.md:5]: This is a duplicate of `.changeset/deprecate-actionlistv1-promote-actionlistv2.md`

File: .changeset/odd-apes-guess.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+"@primer/react": major
+---
+
+`PageLayout` is being graduated from the `drafts` bundle to the `main` bundle.
```
Comment (colebemis) [inline .changeset/odd-apes-guess.md:5]:
```suggestion
---

<br />

### PageLayout

`PageLayout` is being graduated from the `drafts` bundle to the `main` bundle. 
```

File: .changeset/smooth-cameras-prove.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
+---
+'@primer/react': major
+---
+
+### SelectMenu
```
Comment (colebemis) [inline .changeset/smooth-cameras-prove.md:5]:
```suggestion
---

<br />

### SelectMenu
```

**Review by rezrah — APPROVED** [review]
Let's gooo! 🎉

**Pull request merged**